src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1083,6 +1083,8 @@ def validate_azcopy_remove_arguments(cmd, namespace):
     if valid_blob:
         client = blob_data_service_factory(cmd.cli_ctx, {
             'account_name': namespace.account_name})
+        if not blob:
+            blob = ''
```

Comment:
server still need blob to be empty string. right? Is it also necessary for `path`?

Reply:
For `path` part, it already support '' in CLI code. This is a legacy bug from previous member.